recipes/rhash/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+make ADDCFLAGS="${CFLAGS}" ADDLDFLAGS="${LDFLAGS}" build-shared
+
+make PREFIX="" DESTDIR=${PREFIX} install-shared
+
+make -C librhash PREFIX="" DESTDIR=${PREFIX} install-headers install-lib-shared install-lib-static
+
+cd tests
+./test_rhash.sh
```

Comment:
Yes, but maybe they fail because incompatibilities of shell commands rather than incorrect results, lets hear what they have to say about the osx port issue